client/web/web.go

```diff
@@ -291,6 +288,24 @@ func (s *Server) serveTailscaleAuth(w http.ResponseWriter, r *http.Request) {
 
 	session, whois, err := s.getSession(r)
 	switch {
+	case err != nil && errors.Is(err, errNotUsingTailscale):
+		// not using tailscale, so perform platform auth
```

Comment:
I think this works for the login/lobby client, but not the full TS client. In this case it'll say auth is okay if they're not using tailscale as long as the platform auth succeeds. For now, I'd throw a `s.tsDebugMode != "full"` check in here, but my upcoming CLI change adds the new server "Type" field we were taking about, so that should clean this up.

Reply:
in an earlier attempt at this change, I was adding the clientType field to the authResponse as well, and this was the point where I forcibly set the clientType to "login", so it couldn't possibly render the full client when not using tailcale.

Reply:
> but how would you possibly reach the full TS client if you're not using Tailscale? It's only accessible via the Tailscale IP.

Oh whoops, yeah. I mean I guess someone could also specify that they want the full client from a tsnet implementation or something like that?